src/memex/presenters.py

```diff
@@ -6,10 +6,13 @@
 import collections
 import copy
 
+from pyramid import security
+
 
 class AnnotationBasePresenter(object):
-    def __init__(self, annotation, links_service):
-        self.annotation = annotation
+    def __init__(self, annotation_resource, links_service):
```

Comment:
This seems a bit odd to me. Or, to be more accurate, the resulting call pattern looks odd:

```python
annotation = storage.fetch_annotation(id_)
link_service = request.find_service(...)
group_service = request.find_service(...)
resource = AnnotationResource(annotation, group_service)
presenter = AnnotationJSONPresenter(resource, link_service)
```

I think it would be more natural if the presenter accepted the group_service and constructed the resource itself:

```python
annotation = storage.fetch_annotation(id_)
link_service = request.find_service(...)
group_service = request.find_service(...)
presenter = AnnotationJSONPresenter(resource, group_service, link_service)
```